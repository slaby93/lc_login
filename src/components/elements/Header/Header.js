import styled from 'styled-components';
import { prop } from 'styled-tools';

const StyledComponent = styled.h1`
  font-size: ${prop('theme.fontSize.xxbig')}px;
  font-weight: 600;
  margin: 10px;
  color: ${prop('theme.color.black')};
`;

export default StyledComponent;
