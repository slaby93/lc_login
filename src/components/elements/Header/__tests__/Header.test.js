import React from 'react';
import { shallow } from 'enzyme';
import Header from './../Header';

describe('<Header />', () => {
  test('CSS', () => {
    const mountedComponent = shallow(<Header
      className="test"
      placeholder="test"
    >
      TEST
    </Header>);
    expect(mountedComponent).toMatchSnapshot();
  });
});
