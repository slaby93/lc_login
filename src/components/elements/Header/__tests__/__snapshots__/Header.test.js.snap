// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> CSS 1`] = `
.c0 {
  font-size: px;
  font-weight: 600;
  margin: 10px;
}

<h1
  className="test c0"
  placeholder="test"
>
  TEST
</h1>
`;
